### PR TITLE
🌐 Add Indonesian translation for `docs/id/docs/tutorial/path-params.md`

### PR DESCRIPTION
**Pull request by gerry-sabar**

### CODE REVIEW

**Review by RyaWcksn — APPROVED** [review]
LGTM

Approved as a native language

File: docs/id/docs/tutorial/path-params.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+## Parameter path dengan tipe data
+
+Tipe data di parameter path bisa didefinisikan di dalam fungsi, menggunakan anotasi tipe data standar Python:
+
+{* ../../docs_src/path_params/tutorial002.py hl[7] *}
+
+Di kondisi ini `item_id` didefinisikan sebagai `int`.
+
+/// periksa
```
Comment (guspan-tanadi) [inline docs/id/docs/tutorial/path-params.md:23]: are you sure for displaying box these section and Tip section ?

Comment (gerry-sabar) [reply docs/id/docs/tutorial/path-params.md]: @guspan-tanadi Hi, thanks for the review. I have fixed this one

**Comment by gerry-sabar** [comment]
Hi @alejsdev this PR got Labels / check-labels failing. Could you help how to solve it? thanks

**Review by ilhamfadillah — APPROVED** [review]
good translate

**Comment by alejsdev** [comment]
Cool! Thanks for your help @gerry-sabar :nerd_face: :rocket:

**Pull request merged**